pkg/daemon/daemon.go

```diff
@@ -517,6 +517,17 @@ func (dn *Daemon) detectEarlySSHAccessesFromBoot() error {
 func (dn *Daemon) RunOnceFrom(onceFrom string, skipReboot bool) error {
 	dn.skipReboot = skipReboot
 
+	// Unconditionally remove this file in the once-from (classic RHEL)
+	// case.  We use this file to suppress things like kubelet and SDN
+	// starting on CoreOS during the firstboot/pivot boot, but there's
+	// no such thing on classic RHEL.
+	_, err := os.Stat(constants.MachineConfigEncapsulatedPath)
+	if err == nil {
+		if err := os.Remove(constants.MachineConfigEncapsulatedPath); err != nil {
+			return errors.Wrapf(err, "failed to remove %s", constants.MachineConfigEncapsulatedPath)
+		}
+	}
```

Comment:
this should be run after the once from is run - this is a RHEL node so when this run we're effectively writing the encapsulated mc at the end of this, which results, obviously, in the file being there. We should move this at the end.